src/TestApp/NLog.config

```diff
@@ -2,6 +2,9 @@
 <nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xmlns:sl="http://www.nlog-project.org/schemas/NLog.Targets.Syslog.xsd">
+    <extensions>
+      <add assembly="NLog.Targets.Syslog"/>
+    </extensions>
     <variable name="enterpriseId" value="12345"/>
```

Comment:
NET Core places the NuGet-dependencies in seperate folders. So add-assembly is needed for it to work.